librz/core/visual.c

```diff
@@ -3235,7 +3235,8 @@ RZ_API int rz_core_visual_cmd(RzCore *core, const char *arg) {
 				ut64 addr = rz_debug_reg_get(core->dbg, "PC");
 				if (addr && addr != UT64_MAX) {
 					rz_core_seek_and_save(core, addr, true);
-					rz_core_cmdf(core, "ar `arn PC`=0x%" PFMT64x, addr);
```

Comment:
I think this line was written some time ago... you can now do `ar PC=0xcafe` and it works.
I think you can directly do `rz_core_analysis_set_reg(core, "PC", addr)`.

Reply:
Right, changed to the direct call.

Reply:
Right, changed to the direct call.